src/activity-logger/background/log-page-visit.test.js

```diff
@@ -0,0 +1,40 @@
+/* eslint-env jest */
+/* eslint import/namespace: "off" */
+
+import maybeLogPageVisit from 'src/activity-logger/background/log-page-visit'
+import * as storePage from 'src/page-storage/store-page'
+import db from 'src/pouchdb'
+
+afterAll(async () => {
+    await db.destroy()
+})
+
+describe('maybeLogPageVisit', () => {
```

Comment:
Hah, I did exactly this refactoring last night (then to make a manual log-button). The code for the logging flow (including analysis/deduplication) could use more refactoring like this. Please feel free to (suggest to) refactor things while writing tests, as that seems a good moment to discover a more sensible structure.